sign/sign.go

```diff
@@ -47,14 +47,21 @@ type Transaction struct {
 	Body       json.RawMessage `json:"body" swaggertype:"object"` // json string
 }
 
+// returns a sign compatible timestamp for the current wall time
```

Comment:
Go convention is to prefix function/method comment with the function/method name. Otherwise, godoc won't recognize it.

`TimestampNow ...`

Reply:
This would be a good thing for CodeRabbit to pick up on. CodeRabbit, are you listening?

Reply:
@coderabbitai [](edited-with-Axolo)